contrib/TestHarness2/README.md

```diff
@@ -1,88 +1,25 @@
 # FoundationDB TestHarness2
 
-This directory contains TestHarness2, a Python-based test harness for FoundationDB (that supercedes [`../TestHarness`](../TestHarness)), designed to be invoked by the Joshua testing framework via scripts like [`../Joshua/scripts/correctnessTest.sh`](../Joshua/scripts/correctnessTest.sh). In typical FoundationDB testing setups orchestrated by Joshua, this harness and the tests it runs are executed within Kubernetes pods.
+This directory contains TestHarness2, a Python-based test harness for FoundationDB (that supercedes [`TestHarness`](../TestHarness)), designed to be invoked by the [Joshua](../Joshua) testing framework via scripts such as [`correctnessTest.sh`](../Joshua/scripts/correctnessTest.sh). In typical FoundationDB testing setups orchestrated by [Joshua](../Joshua), this harness and the tests it runs are executed within Kubernetes pods.
 
 ## TestHarness2 Operation and Outputs
 
-TestHarness2 outputs individual test results directly to stdout in XML format. Each test that runs produces a single XML summary that is immediately printed to stdout. Joshua consumes these XML outputs from stdout to track test results.
+TestHarness2 outputs individual test results directly to stdout in XML format. Each test that runs produces a single XML summary that is immediately printed to stdout. [Joshua](../Joshua) consumes these XML outputs from stdout to track test results.
```

Comment:
Joshua is not in this repo. Maybe the link should be https://github.com/FoundationDB/fdb-joshua/